src/views/components/application-status.njk

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-grid-row govuk-!-margin-top-4">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l">Application status</h2>
+    <p class="govuk-body">Refresh the page to see updated information. If you exit this page, you can return to it by signing in to <a href="{{ Paths.ROOT_URI }}">the Apply to strike off and dissolve a company service</a>.</p>
+  </div>
+
+  {% set signatoryRows = [] %}
+  {% for signatory in signatories %}
+    {% set signatoryRows =
+      (
+        signatoryRows.push(
```

Comment:
feels like this could be moved into a service to create this object rather than being in the nunjucks and having to be tested by each controller that implements it, rather than being able to mock a sample object out.

Reply:
👍 . Definitely better this is done in a service rather than in the view

Reply:
I've moved some logic into a separate service (name generation) and kept the nunjucks-specific stuff in the template. Template is much simpler now and so are controller unit tests, although still requires this template generation code.